recipes/magic-level/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "magic-level",
+  "name": "Magic Level",
+  "version": "1.25.111",
```

Comment:
If this is a brand new recipe for Ferdium, we should start the version at `1.0.0`